doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc

```diff
@@ -0,0 +1,33 @@
+[id='test-designer-expressions-syntax-dmn-based-ref']
+= Expression syntax in DMN-based scenarios
+The following data types are supported by the DMN-based test scenarios in the *Test Scenario* designer:
```

Comment:
I think it should be useful to mention also here that we support FEEL as language. The rest of the paragraph is just an example of FEEL data types / expressions

Reply:
@danielezonca, in the rendered output, the para just before this section mentions that - "While rule-based test scenarios support a basic expression language, DMN-based test scenarios support the FEEL expression language.". So I believe we should mention the same thing again here. Hence, no change.

Reply:
Ok in this case, fine for me